renderer/locales/id.json

```diff
@@ -0,0 +1,288 @@
+    },
+    "OVERWRITE_TOGGLE": {
+      "TITLE": "TIMPA UPSCALE SEBELUMNYA",
+      "DESCRIPTION": "Jika diaktifkan, Upscayl akan memproses gambar itu lagi alih-alih memuatnya secara langsung."
+    },
+    "RESET_SETTINGS": {
+      "BUTTON_TITLE": "RESET UPSCAYL",
+      "ALERT": "Upscayl telah direset. Silakan restart aplikasi."
+    },
+    "SAVE_OUTPUT_FOLDER": {
+      "TITLE": "SIMPAN FOLDER OUTPUT",
+      "DESCRIPTION": "Jika diaktifkan, folder output akan diingat antara sesi."
+    },
+    "AUTO_UPDATE": {
+      "TITLE": "PEMBARUAN OTOMATIS UPSCAYL",
+      "DESCRIPTION": "Jika diaktifkan, aplikasi akan memeriksa pembaruan baru dan memberi tahu Anda."
+    },
+    "ENABLE_CONTRIBUTION": {
+      "TITLE": "BANTU TINGKATKAN UPSCAYL",
+      "DESCRIPTION": "Jika diaktifkan, Upscayl akan mengumpulkan data penggunaan anonim untuk meningkatkan antarmuka dan fitur aplikasi."
+    },
+    "THEME": {
+      "TITLE": "TEMA UPSCAYL"
+    },
+    "LANGUAGE": {
+      "TITLE": "BAHASA UPSCAYL"
+    },
+    "CUSTOM_TILE_SIZE": {
+      "TITLE": "UKURAN TILE KUSTOM",
+      "DESCRIPTION": "Gunakan ukuran tile kustom untuk membagi gambar. Ini dapat membantu memproses gambar lebih cepat dengan mengurangi jumlah tile yang dihasilkan."
+    },
+    "TURN_OFF_NOTIFICATIONS": {
+      "TITLE": "MATIKAN NOTIFIKASI",
+      "DESCRIPTION": "Jika diaktifkan, Upscayl tidak akan mengirim pemberitahuan sistem apapun tentang keberhasilan atau kegagalan."
+    },
+    "SUPPORT": {
+      "TITLE": "Mengalami masalah?",
+      "DOCS_BUTTON_TITLE": "🙏 MINTA BANTUAN",
+      "EMAIL_BUTTON_TITLE": "📧 EMAIL DEVELOPER"
+    },
+    "TTA_MODE": {
+      "TITLE": "Mode TTA",
+      "DESCRIPTION": "Aktifkan Test Time Augmentation untuk hasil yang lebih baik, seperti menghilangkan artefak TAPI ini akan meningkatkan waktu pengolahan hingga 8x!"
+    },
+    "SYSTEM_INFO": {
+      "TITLE": "Info Sistem"
+    }
+  },
+  "APP": {
+    "TITLE": "Upscayl",
+    "BATCH_MODE": {
+      "TITLE": "Batch Upscayl",
+      "DESCRIPTION": "Ini akan memungkinkan Anda untuk Upscayl semua file dalam folder sekaligus"
+    },
+    "FILE_SELECTION": {
+      "TITLE": "Langkah 1",
+      "BATCH_MODE_TYPE": "Pilih Folder",
+      "SINGLE_MODE_TYPE": "Pilih Gambar"
+    },
+    "MODEL_SELECTION": {
+      "TITLE": "Langkah 2",
+      "DESCRIPTION": "Pilih Model AI",
+      "IMPORTED_CUSTOM_MODELS": "Model Kustom yang Diimpor",
+      "BEFORE": "Sebelum",
+      "AFTER": "Sesudah",
+      "ZOOM": "Perbesar",
+      "MODELS": {
+        "upscayl-standard-4x": {
+          "NAME": "Upscayl Standar",
+          "DESCRIPTION": "Cocok untuk sebagian besar gambar."
+        },
+        "upscayl-lite-4x": {
+          "NAME": "Upscayl Lite",
+          "DESCRIPTION": "Cocok untuk sebagian besar gambar. Upscaling berkecepatan tinggi dengan kehilangan kualitas minimal."
+        },
+        "remacri-4x": {
+          "NAME": "Remacri (Non-Komersial)",
+          "DESCRIPTION": "Untuk gambar alami. "
```

Comment:
Better to use 'natural' for consistency with other.

Reply:
Thanks! Agreed, I'll update it for consistency.